atomic_reactor/plugins/pre_koji.py

```diff
@@ -36,15 +38,33 @@ def __init__(self, tasker, workflow, target, hub, root, proxy=None):
         self.pathinfo = koji.PathInfo(topdir=root)
         self.proxy = proxy
 
+    def _check_target_and_tag(self):
```

Comment:
Not sure, we're reading 'BZComponent' from Dockerfile here, I don't think k-cb can do that, as its the only way to know the component as far as I know